frontend/src/components/creator-module/creator-preview.tsx

```diff
@@ -1,15 +1,45 @@
-import React, { ReactElement } from "react";
+import React, { useState, useEffect, useContext } from "react";
+import { store } from "../../store";
+
 import "./creator-preview.scss";
 
 //TODO: finish creator Preview
 
 //TODO: add props for question
 
-const CreatorPreview = (): ReactElement => {
+interface Prop {
+  newQuestion: Question;
+}
+
+interface Question {
+  title: string;
+  question: string;
+  type: string;
+  choices: string[];
+  correct: number;
+}
+
+const CreatorPreview = ({ newQuestion }: Prop) => {
+  const global = useContext(store) as any;
+  const dispatch = global.dispatch;
+  const state = global.state;
+
+  const previewQuestion = state.editPreviewQuestion
+    ? state.questions[state.previewFolder].questions[state.previewQuestion]
+    : newQuestion;
+
   return (
     <div className="creator-preview">
       <div className="preview-title">
-        <span>{"Question Title Here"}</span>
```

Comment:
Currently this only shows the question and doesn't also show the question title.